src/components/MessageForm.tsx

```diff
@@ -31,18 +31,22 @@ export const MessageForm = ({ confirmAction, cancelAction }: MessageFormProps) =
         <Form noValidate>
           <Field name="message">
             {({ field, meta: { touched, error } }: FieldProps<string>) => (
-              <TextField
-                {...field}
-                disabled={isSubmitting}
-                data-testid="message-field"
-                variant="filled"
-                multiline
-                maxRows={Infinity}
-                fullWidth
-                label={t('common.message')}
-                error={touched && !!error}
-                helperText={<ErrorMessage name={field.name} />}
-              />
+              <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+                <TextField
+                  {...field}
+                  inputProps={{ maxLength: 160 }}
+                  disabled={isSubmitting}
+                  data-testid="message-field"
+                  variant="filled"
+                  multiline
+                  maxRows={Infinity}
+                  fullWidth
+                  label={t('common.message')}
+                  error={touched && !!error}
+                  helperText={<ErrorMessage name={field.name} />}
+                />
+                <span style={{ alignSelf: 'end', color: 'gray' }}>{(field.value as string).length}/160</span>
```

Comment:
Trenger sikkert ikke `as string` her, siden `string` er angitt som type på linje 33.
UU-messig vil jeg anta at det er bedre med Typography også, slik at det blir en `p`-tag i stedet for `span`..? :thinking: 
```suggestion
                <Typography sx={{ alignSelf: 'end', color: 'gray' }}>{field.value.length}/160</Typography>
```